src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int FIGURE_COUNT = 5;
+    private final int sideLength = 5;
```

Comment:
```suggestion
    private final int maxSideLength = 5;
```
\+ плюс цю змінну можна зроибити константою як і FIGURE_COUNT